website/content/docs/lambda/index.mdx

```diff
@@ -6,28 +6,42 @@ description: >-
+After registering AWS Lambda functions, you can invoke Lambda functions from the Consul service mesh through terminating gateways (recommended) or directly from connect proxies.
 
-### Invoking Lambda Functions from Consul Service Mesh
+Refer to Invoke Lambda Functions from Services for details.
 
-Lambda functions can be invoked by any mesh service directly from connect proxies or
-through terminating gateways. The [Invocation page](/docs/lambda/invocation)
-explains how to invoke Lambda functions from Consul service mesh services.
+## Invoke mesh service from Lambda function 
+
+You can also add the `consul-lambda-extension` plugin as a layer in your Lambda functions, which enables them to send requests to services in the mesh. The plugin starts a sidecar proxy that directs requests from Lambda functions to [mesh gateways](docs/connect/gateways#mesh-gateways). The gateways route traffic to the destination service to complete the request.  
+
+![Invoke mesh service from Lambda function](/img/invoke-service-from-lambda-flow.svg)
+
+Refer to [Invoke Services from Lambda Functions](TODO) for additional information about registering Lambda functions into Consul.
+
+## Cross-datacenter communication
+
+You can use the following Consul features to send cross-datacenter requests between Lambda functions and mesh services. 
+
+### Mesh gateway WAN federation
+
+Mesh gateways enable you to route traffic to services within and across Consul datacenters. WAN federation refers to designating a _primary datacenter_ that contains authoritative information about all datacenters, including service mesh configurations and access control list (ACL) resources. Refer to [Mesh Gateways for WAN Federation](/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways) for additional information.
+
+Note that mesh gateways do not implement L7 traffic management by default. As a result, requests from Lambda functions ignore service routes and splitters.
+
+#### Admin partitions 
+
+If admin partitions are enabled and the datacenters are federated across the WAN using mesh gateways, then you can only route requests from Lambda functions by applying an [`exported-services`](/docs/connect/config-entries/exported-services) configuration entry to export their service instances. This is required even if the upstream for the Lambda function is in the same admin partition. Otherwise, Consul does not populate the mesh gateways with the routing information.
+
+You can also use the [admin partitions](/docs/enterprise/partitions) feature included with Consul Enterprise to define separate administrative areas within a datacenter. If admin partitions are not enabled and the datacenters are federated across the WAN using mesh gateways, then you can route all services through the mesh gateways by default. You do not need to use the [`exported-services`](/docs/connect/config-entries/exported-services) configuration entry to export service instances. 
```

Comment:
Just to be clear, I need to remove the language around WANFed and APs in this section and say that the user just needs to implement a mesh gateway? Is the behavior/configuration the same, whether or not the DCs are federated? Whether or not APs are enabled?

Reply:
Yeah, the behavior is the same. If WAN fed is being used, the mesh gateway can forward requests to another DC. If APs are being used, it can forward requests to another AP. Without either, it can only send requests to the same DC.